src/mudlet.cpp

```diff
@@ -584,11 +584,26 @@ mudlet::mudlet()
     mpActionVariables->setObjectName(QStringLiteral("variables_action"));
     mpMainToolBar->widgetForAction(mpActionVariables)->setObjectName(mpActionVariables->objectName());
 
-    mpActionIRC = new QAction(QIcon(QStringLiteral(":/icons/internet-telephony.png")), tr("IRC"), this);
-    mpActionIRC->setToolTip(tr("Open the Mudlet IRC client"));
-    mpMainToolBar->addAction(mpActionIRC);
-    mpActionIRC->setObjectName(QStringLiteral("irc_action"));
-    mpMainToolBar->widgetForAction(mpActionIRC)->setObjectName(mpActionIRC->objectName());
+    mpButtonDiscord = new QToolButton(this);
+    mpButtonDiscord->setText(tr("Discord"));
```

Comment:
Maybe put a more generic "Chat" here, to also include the IRC

Reply:
Or not?

Reply:
Why? This button does not open "chat", it opens Discord

Reply:
I was thinking this like
* Discord is a name of one service to chat with others
* Chat is the activity you do and are interested in here
* Not everybody understands what "Discord" means
* This button is also for dropdown which includes IRC
* IRC has gone from main menu, people may search

Actually I suggested alike back in 2017 already 😁  https://github.com/Mudlet/Mudlet/issues/1387#issuecomment-343154394

Reply:
Let's try it out in the field and see what player feedback is like - if people are confused, we'll fix. I suspect this is more of a theoretical problem one than a practical, one, but I could be wrong.